draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -236,31 +236,31 @@ RADIUS/(D)TLS client implementations SHOULD implement this model, but MUST imple
-  * For clients configured by their source IP address, the configured IP address is matched against the presented addresses in the subjectAltName:iPAddr extension; if no such exist, against the presented CN component of the certificate subject.
-    For clients configured by IP range, the certificate MUST be valid for the IP address the client is currently using.
-  * It is possible for a RADIUS/(D)TLS server to not require additional name checks for incoming RADIUS/(D)TLS clients, i.e. if the client used dynamic lookup.
-    In this case, the certificate is accepted immediately after the {{RFC5280}} trust chain checks.
-    This MUST NOT be used outside of trusted network environments or without additional certificate attribute checks in place.
-* Implementations MAY allow a configuration of a set of additional properties of the certificate to check for a peer's authorization to communicate (e.g. a set of allowed values in subjectAltName:URI or a set of allowed X.509v3 Certificate Policies).
-* When the configured trust base changes (e.g., removal of a CA from the list of trusted CAs; issuance of a new CRL for a given CA), implementations SHOULD renegotiate the TLS session to reassess the connecting peer's continued authorization.[^may-should-trustbase]{:jf}
+* When the configured trust base changes (e.g., removal of a CA from the list of trusted CAs; issuance of a new CRL for a given CA), implementations SHOULD reassess all connected peer's continued authorization. Note that TLS 1.3 no longer supports renegotiation to fulfill this requirement. [^may-should-trustbase]{:jf}
 
 [^may-should-trustbase]: Open discussion: RFC6614 says "may" here. I think this should be a "should". There are some discussions to change this to "must". Input from TLS/UTA experts is appreciated.
 
+RADIUS/(D)TLS clients and server MUST follow {{!RFC9525}} when validating peer identities. Specific details are provided below:
+* The Common Name RDN MUST NOT be used to identify peers
+* Certifcates MAY use wildcards in the identifiers of DNS names and realm names, but only as the complete, left-most label.
+* RADIUS/(D)TLS clients validate the servers identity to match their local configuration, accepting the identity on the first match:
+  - If the expected RADIUS/(D)TLS server is associated with a specific NAI realm, e.g. by dynamic discovery {{!RFC7585}} or static configuration, that realm is matched against the presented realm names in the subjectAltName:naiRealm extension.
+  - If the expected RADIUS/(D)TLS server was configured as a hostname, or yielded by a dynamic discovery procedure, the configured name is matched against the presented names from the subjectAltName:DNS extension. Since a dynamic discovery might by itself not be secured, implementations MAY require the use of DNSSEC {{!RFC4033}} to ensure the authenticity of the DNS result before considering this identity as valid. 
```

Comment:
```suggestion
  - If the expected RADIUS/(D)TLS server was configured as a hostname, or the hostname was yielded by a dynamic discovery procedure, the configured name is matched against the presented names from the subjectAltName:DNS extension. Since a dynamic discovery might by itself not be secured, implementations MAY require the use of DNSSEC {{!RFC4033}} to ensure the authenticity of the DNS result before considering this identity as valid. 
```

Just for clarity.